packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -511,11 +511,17 @@ export class DocumentDeltaConnection
     protected createErrorObject(handler: string, error?: any, canRetry = true): DriverError {
         // Note: we suspect the incoming error object is either:
         // - a string: log it in the message (if not a string, it may contain PII but will print as [object Object])
-        // - a socketError: add it to the OdspError object for driver to be able to parse it and reason
-        //   over it.
-        let message = `socket.io: ${handler}`;
-        if (typeof error === "string") {
+        // - an Error object thrown by socket.io engine. Be careful with not recording PII!
+        let message = `socket.io (${handler})`;
+        if (typeof error !== "object") {
             message = `${message}: ${error}`;
+        } else if (error?.type === "TransportError") {
+            // Websocket errors reported by engine.io-client.
+            // They are Error objects with description containing WS error and description = "TransportError"
+            // Please see https://github.com/socketio/engine.io-client/blob/master/lib/transport.js#L30,
```

Comment:
Link was 404 for me.  Perhaps:
```
https://github.com/socketio/engine.io-client/blob/master/lib/transport.ts#L44
```

Reply:
Maybe one of these would be more stable as opposed to linking to master:
https://github.com/socketio/engine.io-client/blob/7245b803e0c8d57cfc1f1cd8b8c8d598e8397967/lib/transport.ts#L44
https://github.com/socketio/engine.io-client/blob/3.5.0/lib/transport.js#L69